src/jobs/withdrawalProcessingJob.ts

```
@@ -2,7 +2,7 @@
  * Consumes WITHDRAWAL_PROCESSING queue: after BurnEvent, validate withdrawal and recipient,
  * disburse via fintech, update transaction status, optionally publish user notification.
  */
-import { randomUUID } from "crypto";
+import { generateId } from '../utils/idGenerator';
```

ESLint

[error] Replace `'../utils/idGenerator'` with `"../utils/idGenerator"`

(prettier/prettier)